main.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/gorilla/mux"
+)
+
+type Light struct {
+	ID     string `json:"id"`
+	Name   string `json:"name"`
+	Status string `json:"status"`
+}
+
+var db *sql.DB
+var err error
+
+func main() {
+	db, err = sql.Open("mysql", "root@localhost:123asy123asy@tcp(127.0.0.1:3306)/lights_api")
+	if err != nil {
+		panic(err.Error())
+	}
+
+	defer db.Close()
+
+	router := mux.NewRouter()
+
+	router.HandleFunc("/lights", getLights).Methods("GET")
+	router.HandleFunc("/lights/create", createLights).Methods("POST")
+	router.HandleFunc("/lights/update", updateLights).Methods("POST")
+	router.HandleFunc("/lights/delete", deleteLights).Methods("POST")
```

Comment:
Shouldn't this be DELETE method?

Reply:
Okay! Changed to Delete method!
Would be updated with the next commit!